tests/displaySearchedSockTest.php

```diff
@@ -0,0 +1,151 @@
+<?php
+require_once 'src/functions.php';
+
+use PHPUnit\Framework\TestCase;
+
+class displaySearchedSockTest extends TestCase {
+
+    public function testDisplaySearchedSockSuccessTest() :void
+    {
+        $array = [
+            'Size' => 'XS',
+            'Name' => 'name',
+            'Pattern' => 'zigzag',
+            'Color' => 'red',
+            'Description' => 'description'
+        ];
+        $int = 1;
+
+        $expected = '<br>';
+        $expected .= "<p><b>Size:</b> XS</p>";
+        $expected .= "<p><b>Name:</b> name</p>";
+        $expected .= "<p><b>Pattern:</b> zigzag</p>";
+        $expected .= "<p><b>Color:</b> red</p>";
+        $expected .= "<p><b>Description:</b> description</p>";
+        $expected .= "<br><form method=\"post\">";
+        $expected .= "<label for=\"deletesock\"></label>";
+        $expected .= "<input name=\"deletesock\" id=\"deletesock\" value=\"1\">";
+        $expected .= "<input class=\"submit\" type=\"submit\" value=\"DELETE SOCK\">";
+        $expected .= "</form>";
```

Comment:
Could tidy this up by not repeating "$expected" on each line, makes for easier reading.